src/app/documentation-section/documentation-section.component.scss

```diff
@@ -0,0 +1,19 @@
+:host {
+	h3 {
+		margin-top: 0;
+	}
+
+	.hide {
+		display: none;
+	}
+
+	.section {
+		max-width: 100%;
+		padding: 1em;
+	}
+
+	// TODO: remove after markdown component is created
```

Comment:
This should probably be removed after testing to make sure that it, in fact, is no longer needed now that the markdown component exists.